Cannot test new .opam packages, it fails to install

Hi,

I am trying to transition ounit OPAM package to ounit2. To do that, I have bot ounit.opam and ounit2.opam and they depend on each other (precise version).

The problem is that it fails to install, because "ounit2" package cannot be found:
```
opam depext -u ounit
+opam depext -u ounit
# Detecting depexts using vars: arch=x86_64, os=linux, os-distribution=ubuntu, os-family=debian
[ERROR] No solution for ounit: The following dependencies couldn't be met:
          - ounit → ounit2 >= 2.1.2
              no matching version
```
The [full logs](https://travis-ci.org/gildor478/ounit/jobs/589294999)

I have already added PINS="ounit2:. ounit:.".

I know that once the package will be published, there will be no more problem -- but I would like to test before sending it to OPAM repository.

Any ways to solve the problem with ocaml-ci-scripts?

--- a/README-travis.md
+++ b/README-travis.md
@@ -192,6 +192,18 @@ that package being pinned to the URL in that package's `dev-repo`
 field. A pin of a `name:url` or `name.version:url` pair will pin the
 package to the given URL.
 
+### Building with new and not yet published packages
+
+```yaml
+env:
+  - [...] PINS="ounit2.2.2.0:. ounit2-lwt.2.2.0:." [...]
+```
+
+If you start a project from scratch and use these scripts without a
+pre-existing package in OPAM, you need to pin the package to a precise
+version directly in the environment. This will allow to install the
+package and other dependencies correctly.
+
 ### Hooks
 
 ```yaml

